contributor/reference/stable-release.rst

```diff
@@ -0,0 +1,76 @@
+Stable release process
+======================
+
+Key Steps at Each Stage
+-----------------------
+
+Rocks
++++++
```

Comment:
H3 should be represented this way (see [style guide](https://canonical-starter-pack.readthedocs-hosted.com/latest/reference/style-guide/) for info on how to represent H1 through H5:

Heading
~~~~~~~